client/state/domains/management/actions.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_RECEIVE,
+	DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_REQUEST,
+	DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_REQUEST_FAILURE,
+	DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_REQUEST_SUCCESS,
+	DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_UPDATE,
+	DOMAIN_MANAGEMENT_WHOIS_RECEIVE,
+	DOMAIN_MANAGEMENT_WHOIS_REQUEST,
+	DOMAIN_MANAGEMENT_WHOIS_REQUEST_FAILURE,
+	DOMAIN_MANAGEMENT_WHOIS_REQUEST_SUCCESS,
+	DOMAIN_MANAGEMENT_WHOIS_SAVE,
+	DOMAIN_MANAGEMENT_WHOIS_SAVE_FAILURE,
+	DOMAIN_MANAGEMENT_WHOIS_SAVE_SUCCESS,
+	DOMAIN_MANAGEMENT_WHOIS_UPDATE,
+} from 'state/action-types';
+
+/**
+ * Returns an action object to be used in signalling that a cached domains
+ * contact details object has been received.
+ *
+ * @param   {Object}   cacheData	cached contact details object
+ * @returns {Object}   Action object
+ */
+export function receiveContactDetailsCache( cacheData ) {
+	return {
+		type: DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_RECEIVE,
+		cacheData
```

Comment:
We usually call it `payload` in line with the [Flux Standard Action](https://github.com/acdlite/flux-standard-action)